src/components/Square.js

```diff
@@ -0,0 +1,47 @@
+// - User story: I can navigate to localhost:3000 in the browser and see a React application. √
+// - User story: The app has a header and a square on the screen with the word "off" in the middle. √
+// - User story: When I click the square the word toggles from "off" to "on" and then back to "off" again. √
+// - User story: When the lightbulb is "off" the background color of the box is white. When the lightbulb is "on" the background color of the box is yellow. √
+
+// I am assuming the 'lightbulb' referred to in the 4th user story is the square and that it'll turn into a lightbulb later
+
+import React, { Component } from "react";
+
+export default class Square extends Component {
+  // exports and creates class syntax
+  constructor(props) {
+    super(props);
+    this.state = {
+      offOn: ["OFF", "ON"], // array of the two text options
+      backgroundColors: ["White", "Yellow"], // array of the two background options
+      index: 0, // to be altered by set state, currently 0 wich means offOn[index] = "OFF" and backgroundColors[index] = "White"
+    };
+  }
+
+  // sets the state of index to either 0 or 1
+  changeIndex = () => {
+    this.setState({
+      index:
+        this.state.index === 0 // if it already equals 0
+          ? (this.state.index = 1) // then change it to 1
+          : (this.state.index = 0), // otherwise change it to 0
+    });
```

Comment:
I think the syntax for the ternary looks a little easier to understand if the ? was on the line that the true/false statement is on and then the : on the line with the return for true is on. This is mainly for readability and doesn't affect operation. Otherwise looks great.

Reply:
Thanks @gitgood-ENT, i agree, but i have that Prettier plug in installed and it's doing it automatically, i'm not sure how to change it!